modules/getting-started/pages/check-published-api-specs.adoc

```diff
@@ -1,7 +1,7 @@
-= Check Published RAML API Specifications Starting Soon
+= Check Published RAML API Specifications Starting January 10, 2019
 
-Soon, the API editor in Design Center will be more precise in ensuring that the RAML in your API specifications is valid. However, it will be possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange will include invalid RAML that those previous versions did not flag.
+As of January 10, 2019, the API editor in Design Center is more precise in ensuring that the RAML in your API specifications is valid. However, it will be possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange will include invalid RAML that those previous versions did not flag.
```

Comment:
Difficult to figure this long sentence: "However, it will be possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange will include invalid RAML that those previous versions did not flag."
How about:
"However, it's possible that one or more of your APIs was created or published with a previous version of the API editor. These APIs can contain invalid RAML that previous versions of the editor did not flag."